super32emu/super32emu/logic/emulator.py

```diff
@@ -165,7 +170,15 @@ def __arithmetic_instruction(self, first_source: str, second_source: str, target
         elif func == self.commands['arithmetic']['OR']:
             result = r1_value | r2_value
         elif func == self.commands['arithmetic']['NOR']:
-            result = r1_value | r2_value ^ 0b11111111
+            result = r1_value | r2_value ^ 0xffffffff
+        elif func == self.commands['arithmetic']['NAND']:
+            result = r1_value & r2_value ^ 0xffffffff
```

Comment:
```suggestion
            result = (r1_value & r2_value) ^ 0xffffffff
```

Not needed here, just makes it easier to read